source/gui/settingsDialogs.py

```diff
@@ -1176,6 +1197,12 @@ def makeSettings(self, settingsSizer):
 	def postInit(self):
 		self.detectFormatAfterCursorCheckBox.SetFocus()
 
+	def onReportIndent(self, evt):
+		if self.lineIndentationCheckBox.GetValue():
+			self.indentTypeList.Enable(True)
+		else:
+			self.indentTypeList.Enable(False)

```

Comment:
.Enabled is a property, so you can get/set it. Under the hood, it calls Enable(). Either way is fine; I just find properties nicer to read personally, but no need to change this now in your new code.